example/kustomization.yaml

```diff
@@ -38,7 +38,7 @@ resources:
 # Notebook Controller
 - ../apps/jupyter/notebook-controller/upstream/overlays/kubeflow
 # Profiles + KFAM
-- ../apps/profiles/upstream/overlays/kubeflow
+# - ../apps/profiles/upstream/overlays/kubeflow
```

Comment:
Are there any reason to remove Profile controller from Kubeflow example installation?